components/ServiceCard/index.js

```diff
@@ -0,0 +1,38 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import useSWR from "swr";
+
+export default function ServiceCard() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { data, isLoading, error } = useSWR(`/api/services/${id}`);
```

Comment:
`data` isn't very meaningful. I recommend you to rename this variable. If you accept this suggestion, please remember to rename the variable in the whole file. 

```suggestion
  const { data: services, isLoading, error } = useSWR(`/api/services/${id}`);
```